### PR TITLE
Publish articles on OpenTelemetry w/ Next.js

### PR DESCRIPTION
**Pull request by ccschmitz**
## Summary

Adds two blog posts from @cmgriffing on the Highlight blog. Originally written as one article and broken up into a generic intro to OTEL article and OTEL/Highlight in Next.js.

Also updates the next config to allow images from all external sources and the absolute image paths (for OG images) to use `NEXT_PUBLIC_VERCEL_PROJECT_PRODUCTION_URL` instead of `NEXT_PUBLIC_VERCEL_URL`, which references `highlight.io` instead of the branch build URL.

## How did you test this change?

You can click test in the Vercel preview for the branch.

## Are there any deployment considerations?

N/A

## Does this work require review from our design team?

N/A

### CODE REVIEW

File: blog-content/how-to-use-opentelemetry-to-monitor-nextjs-apps.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
+---
+title: How to Use OpenTelemetry to Monitor Next.js Applications
+createdAt: 2024-07-26T12:00:00Z
+readingTime: 10
+authorFirstName: Chris
+authorLastName: Schmitz
```
Comment (ccschmitz) [inline blog-content/how-to-use-opentelemetry-to-monitor-nextjs-apps.md:6]: Need to update to Griffing

File: blog-content/how-to-use-opentelemetry-to-monitor-nextjs-apps.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+createdAt: 2024-07-26T12:00:00Z
+readingTime: 10
+authorFirstName: Chris
+authorLastName: Griffing
+authorTitle: Technical Developer Advocate
+authorTwitter: 'https://x.com/cmgriffing'
+authorLinkedIn: 'https://www.linkedin.com/in/cmgriffing'
+authorGithub: 'https://github.com/cmgriffing'
+authorWebsite: 'https://chrisgriffing.com'
+authorPFP: 'https://chrisgriffing.com/static/3053e1631c1e5e072c0c872f7c59f801/6e63d/selfportrait.jpg'
```
Comment (ccschmitz) [inline blog-content/how-to-use-opentelemetry-to-monitor-nextjs-apps.md:12]: This image appears broken on the article view.

File: blog-content/how-to-use-opentelemetry-to-monitor-nextjs-apps.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
+---
+title: How to Use OpenTelemetry to Monitor Next.js Applications
+createdAt: 2024-07-26T12:00:00Z
```
Comment (ccschmitz) [inline blog-content/how-to-use-opentelemetry-to-monitor-nextjs-apps.md:3]: @Vadman97 have any thoughts about how/when you want to roll these out?

Comment (Vadman97) [reply blog-content/how-to-use-opentelemetry-to-monitor-nextjs-apps.md:3]: discussed offline, any dates work here

File: highlight.io/public/images/blog/how-to-use-opentelemetry-to-monitor-nextjs-apps/highlight-session.gif
Comment (Vadman97) [inline highlight.io/public/images/blog/how-to-use-opentelemetry-to-monitor-nextjs-apps/highlight-session.gif:1]: sick gif, what did you make this with?

**Review by Vadman97 — APPROVED** [review]

File: blog-content/what-is-opentelemetry.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+authorTwitter: 'https://x.com/cmgriffing'
+authorLinkedIn: 'https://www.linkedin.com/in/cmgriffing'
+authorGithub: 'https://github.com/cmgriffing'
+authorWebsite: 'https://chrisgriffing.com'
+authorPFP: 'https://chrisgriffing.com/static/3053e1631c1e5e072c0c872f7c59f801/6e63d/selfportrait.jpg'
+tags: OpenTelemetry
+metaTitle: Learn about OpenTelemetry and how it can give you superpowers monitoring your web applications allowing to find and fix bugs and performance problems easily.
+---
+
+OpenTelemetry is a really cool project from the [CNCF](https://www.cncf.io/). In this post, we will cover the origins of OpenTelemetry, also known as OTEL, as well as what it looks like to spin up your own collector. By the end of it, you should have a pretty good understanding about what OpenTelemetry is and how to start collecting OTEL data in your applications. In a future post we'll cover (TODO: link to other article) instrumenting a Next.js app how you can use Highlight to simplify the process while getting additional insights.
```
[the remark this answers is not in the document]
Comment (ccschmitz) [reply blog-content/what-is-opentelemetry.md]: Ah, my bad - thanks!

File: blog-content/what-is-opentelemetry.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+If you would prefer a video of this content, please feel free to check out our YouTube video about the subject.
+
+<iframe width="100" height="100" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen title="What is OpenTelemetry?" src="https://youtu.be/ASgosEzG4Pw"></iframe>
```
[the remark this answers is not in the document]
Comment (ccschmitz) [reply blog-content/what-is-opentelemetry.md]: Updated!

File: highlight.io/public/images/blog/how-to-use-opentelemetry-to-monitor-nextjs-apps/highlight-session.gif
Comment (ccschmitz) [reply highlight.io/public/images/blog/how-to-use-opentelemetry-to-monitor-nextjs-apps/highlight-session.gif:1]: I used https://zight.com

**Comment by Vadman97** [comment]
> @Vadman97 looks like those changes to the next config to allow external images from all sources + the change to reference `NEXT_PUBLIC_VERCEL_PROJECT_PRODUCTION_URL` instead of `NEXT_PUBLIC_VERCEL_URL` worked out. I'm seeing all the images now and the og images still work + hit the `highlight.io` domain instead of the build URL.

awesome! thanks for figuring that unrelated bug out

**Pull request merged**